docs/reference/contributing/guidelines/workflow.md

```diff
@@ -219,11 +219,29 @@ We use many other labels to summarize the scope and effect of the changes.
 
 - *needs: preceding PR* - This pull request cannot yet be merged because it has a dependency on another pull request that needs to merge first.
 - *DO NOT MERGE* - This pull request contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
-- *devices: 'name'* - The pull request specifically affects the named device(s).
-- *component: 'name'* - The pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
+- *devices: 'name'* - This pull request specifically affects the named device(s).
+- *component: 'name'* - This pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
+- *rollup PR* - This pull request is ready for CI testing, but will instead be brought into a rollup pull request to test multiple pull requests at once.
 
 The following labels summarize the scope of the pull request.
 
 - *scope: bug-fix*.
 - *scope: feature*.
 - *scope: new-target*.
+
+#### Rollup pull requests
+
+When Mbed OS has many small, orthogonal pull requests waiting for CI testing to start, maintainers have the option of putting multiple pull requests through CI by creating a rollup pull request. Rollup pull requests are generated by merging other pull requests into a temporary branch, and opening a new pull requests against this new branch. Once the rollup pull request passes CI testing and is merged, the original pull requests that were used to build the rollup pull requests are also automatically merged.
+
+When a pull request is selected to be integrated into a rollup pull requests, it already has a release label and is patiently waiting to start CI testing. Once selected to be looped into a rollup pull request, the original pull request will gain a *rollup PR* label. When the rollup pull request is generated, a comment will also be added to the original pull request mentioning that no further work needs to be done to it. 
```

Comment:
"a comment will also be added to the original pull request mentioning that no further work needs to be done to it"
this comment may be confusing, particularly if the PR later needs to be removed from the rollup-PR in which case it will actually require more work. I think what you may want to convey is that while the PR is bein processed as part of the rollup no further work can be done on the original PR (otherwise the rollup will need to be re-generated).

Reply:
But that's not actually true. If a PR that was put into a Rollup PR is updated on its own, work _shouldn't_ be needed in the original PR. However, if an additional commit shows up in the original PR, and the Rollup PR completes CI and is merged, the updated PR will remain open because not all commits were in the Rollup PR.

We actually had this happen with one of the early tests of the Rollup PR, and fortulately nothing critical broke :)